test/morgan.js

```diff
@@ -1067,7 +1219,7 @@ describe('morgan()', function () {
           var cb = after(2, function (err, res, line) {
             if (err) return done(err)
             var masked = line.replace(/\x1b\[(\d+)m/g, '_color_$1_')
-            assert.equal(masked, '_color_0_GET / _color_0_- _color_0_- ms - -_color_0_')
```

Comment:
Removed them due to the change in the actual responses when using 'dev' after changes.  Since it was dried up the response no longer is wrapped in the color resets, because the default tokens do not have them.  

So, a couple options, I can rollback the the changes to dev, or add a color reset to all functions via the compile(which may break all current response test), or leave it as is, due to the now inserted token having its own resets on front and back for all returned values.  

Appreciate any advice.